HPXMLtoOpenStudio/resources/data/schedules_affected.csv

```diff
@@ -0,0 +1,36 @@
+Schedule Name,Affected By Vacancy,Affected By Power Outage
+occupants,true,false
+lighting_interior,true,true
+lighting_exterior,true,true
+lighting_garage,true,true
+lighting_exterior_holiday,true,true
+cooking_range,true,true
+refrigerator,false,true
+extra_refrigerator,false,true
+freezer,true,true
```

Comment:
Seems like this should be false, true like a refrigerator.

Reply:
Good catch.